repose-aggregator/tests/performance-tests/src/performanceTest/scala/filters/openapivalidator/OpenApiValidatorFilterSimulation.scala

```diff
@@ -55,26 +55,38 @@ class OpenApiValidatorFilterSimulation extends AbstractReposeSimulation {
       )
     }
 
+  var feederRole: Array[Map[String, Any]] =
```

Comment:
No idea why these are `var`s. I think I copied this pattern when I wrote this simulation.

Also, a feeder just has to be of type `Iterator[Map[String, T]]`:
https://gatling.io/docs/current/session/feeder

So really, if you make `ScenariosRole` a map and want to rely on implicits, this could just be:
```
val feederRole = Array(ScenariosRole)
```

Or (I think) you could drop the reliance on an implicit conversion and do:
```
val feederRole = Iterator.continually(ScenariosRole)
```